lib/Support/IRHelpers.cpp

```diff
@@ -345,11 +345,9 @@ findJumpTarget(llvm::BasicBlock *&Result,
                  "This block leads to multiple jump targets");
     Result = BB;
   } else {
-    for (BasicBlock *Predecessor : predecessors(BB)) {
-      if (Visited.count(Predecessor) == 0) {
+    for (BasicBlock *Predecessor : predecessors(BB))
+      if (!Visited.contains(Predecessor))
```

Comment:
Please double check again all the `!`, `!=` and friends.
I want to be sure we don't f- this up and introduce super-subtle bugs.

Reply:
I went through them three times, but sounds good, I'll do one extra pass-through right before the merge.

Reply:
Did an extra look-through.